docs/runbook.md

```diff
@@ -13,103 +13,195 @@
   - [tags](#tags)
   - [parent](#parent)
     - [path](#path)
-    - [strategy](#strategy)
   - [extension](#extension)
   - [variable](#variable)
     - [is_secret](#is_secret)
     - [file](#file)
     - [name](#name-1)
-  - [value](#value)
-  - [artifact](#artifact)
-    - [name](#name-2)
-    - [locations](#locations)
-      - [type](#type)
-      - [path](#path-1)
+    - [value](#value)
   - [notifier](#notifier)
     - [console](#console)
+      - [log_level](#log_level)
     - [html](#html)
+      - [path](#path-1)
+      - [auto_open](#auto_open)
   - [environment](#environment)
-    - [name](#name-3)
-    - [topology](#topology)
-    - [nodes](#nodes)
-    - [nodes_requirement](#nodes_requirement)
-      - [type](#type-1)
+    - [environments](#environments)
+        - [name](#name-2)
+        - [topology](#topology)
+        - [nodes](#nodes)
+        - [nodes_requirement](#nodes_requirement)
+          - [type](#type)
   - [platform](#platform)
   - [testcase](#testcase)
+    - [criteria](#criteria)
 
 ## How-to
 
 ### Configure Azure deployment
 
+Start with lisa\microsoft\runbook\azure.yml, refer this sample yml file to deploy nodes on Azure.
+
 ### Select and set test cases
 
+[testcase](#testcase) section is used to select test cases, we can select the cases by specify 'area', 'category', 'priority', 'tags' in criteria, specify 'include', 'exclude', 'forceInclude', 'forceExclude' for select action.
+
 ### Use variable and secrets
 
+Using variable and secrets is to avoid updating many places for the same value and pass variable value from command line, keep sensitive data shown with mask.
+
 ### Use partial runbook
 
+With [parent](#parent) section, we can reuse the parent yml file which contains the common use definition.
```

Comment:
it needs at least two runbooks to understand it. And it needs to introduce how the relative path works, and how the variables works.